app/config/translations.json

```diff
@@ -126,13 +141,19 @@
     "Fin Source": "Funds"
   },
   "es": {
```

Comment:
Maybe we should move all translation files to separate files, and combine the files as part of the build process?